### DISCUSSION TITLE
How scoop deal with linux-only software and applications?

### DISCUSSION THREAD

**Discussion by LinArcX** [ScoopInstaller/Scoop#5789] [General]
I can see some packages already exist in the repos like Gedit.

I have some questions:
1. How did Scoop build this software? Does it use msys2 or Cygwin or something else?
1. If it's possible to build Linux-only software, can we request for them? (I have lots of packages in my mind, like gf2, bvi, etc..)

**Comment by rashil2000** [comment]
Gedit is available for Windows in a pre-built form, that's we we can package it. 

Scoop does not build packages - it only downloads and installs them.
